src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import { User } from '../../types/User';
+
+type Props = {
+  todoUser: User | null | undefined;
+};
+
+export const UserInfo:React.FC<Props> = ({ todoUser }) => {
+  const { email, name } = todoUser as User;
```

Comment:
It looks like it causes errors in tests
So in this case probably better not to use restructuring (taking into account that `todoUser` could be `null`/`undefined`)

Reply:
sorry for confusing